### PR TITLE
Feat/spanish localization

### LINKED ISSUE #6582: Spanish Translation
### What would you like to be added

Hi, I would like to translate the Kubernetes dashboard into Spanish.

### Why is this needed

Increase the reach of the product into the Hispanic community

### Comments

None

### PR DESCRIPTION
**Pull request by anyulled**
Spanish translation of the Dashboard UI

Followed instructions on [Internationalization](https://github.com/kubernetes/dashboard/blob/master/docs/developer/internationalization.md)

resolves #6582

### CODE REVIEW

File: i18n/locale_conf.json [hunk 1]
```diff
@@ -1 +1 @@
-{"translations": [ "de", "en", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
+{"translations": [ "de", "es","en", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
```
Comment (shu-mutou) [inline i18n/locale_conf.json:1]:
```suggestion
{"translations": [ "de", "en", "es", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
```
Sort alphabetically.

**Comment by shu-mutou** [comment]
@raelga, could you rebase and add @anyulled to OWNERS file?
@anyulled, could you review `i18n/es/messages.es.xlf`?

**Comment by anyulled** [comment]
hi, @shu-mutou I'm afraid I don't have an approver role in the project, I added myself in the past and the @k8s-ci-robot added the label `do-not-merge/invalid-owners-file`.

I'll try in the future if my role gets upgraded

**Comment by shu-mutou** [comment]
You have been already k8s community member, so I think it's ok to add yourself to OWNERS file. Could you try it again?

**Comment by anyulled** [comment]
> You have already been a k8s community member, so I think it's ok to add yourself to the OWNERS file. Could you try it again?

It worked! thanks, now we're missing the LGTM & approved labels and we're good to go.

**Comment by gamba47** [comment]
@anyulled looks fine to me.

File: i18n/es/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+approvers:
+  - raelga
+
```
Comment (shu-mutou) [inline i18n/es/OWNERS:3]: Multiple approvers are preferred because the approver can't approve yourself.

Comment (shu-mutou) [reply i18n/es/OWNERS:3]: Also, dashboard maintainer can not review each languages.

Comment (raelga) [reply i18n/es/OWNERS:3]: @electrocucaracha mind joining as approver?

**Comment by shu-mutou** [comment]
Some new untranslated words have been added into `i18n/messages.xlf` during review.
Could you merge them into `i18n/es/messages.es.xlf ` using `npm fix` and translate them?

**Comment by shu-mutou** [comment]
Before conflict occur, merge this once.
Please keep to invite reviewers and update your translations in the new PR.

**Comment by anyulled** [comment]
Many thanks @shu-mutou !

**Pull request merged**